doc/release-notes/4597-fix-invalid-identify-response.md

```diff
@@ -0,0 +1 @@
+Added the following database options: Scheme, RepositoryIdentifier, Delimiter, SampleIdentifier.
```

Comment:
Can you please change all these (in the code too, of course) to start with "Harvesting"? That is:

:HarvestingScheme, :HarvestingRepositoryIdentifier, :HarvestingDelimiter, :HarvestingSampleIdentifier

Otherwise, they're too generic, especially :Scheme or :Delimiter.

Reply:
Whoops, I jumped the gun. Can you please have these start with "Oiapmh" instead? Like this:

:OaipmhScheme, :OaipmhRepositoryIdentifier, :OaipmhDelimiter, :OaipmhSampleIdentifier

Also, can you please document these in doc/sphinx-guides/source/installation/config.rst? Thanks!